AlgebraInLean/Chapter01/Sheet04.lean

```diff
@@ -59,9 +59,11 @@ instance : AbelianGroup ℤ where
 Structures other than numbers also have the group properties. Take the rotational symmetries of a
 regular polygon, for example. Each element could be a rotation that maintains the symmetries of the
 polygon, and the group operation can be composition of these rotations. For example, consider the
-rotations of such a triangle. All symmetric rotations must be multiples of 120°, of which there are
-three: 0°, 120°, and 240°. Any other rotations can be written as one of these three and some amount
-of 360° rotations. Now, we can prove that this set, commonly called C₃, is a group.
+rotations of a triangle. All symmetric rotations must be multiples of 120°, of which there are
+three: 0°, 120°, and 240°. Any other rotations can be written as one of these three composed with an
+arbitrary amount of 360° rotations. We consider a rotation of 120° and a rotation of 480° to be
+equivalent (i.e. they are the same element of the group), for example, because they are off by a
```

Comment:
Maybe say "arbitrary number" and say "their difference is a multiple of 360"?

Reply:
Yes, I agree. That is more precise